common/spec/dependabot/pull_request_creator/message_builder_spec.rb

```diff
@@ -2649,6 +2649,55 @@ def commits_details(base:, head:)
             )
           end
         end
+
+        context "with five dependencies in one directory and less then 5 in another" do
```

Comment:
I think we should rename this test to something that captures what this PR is for

maybe "with a table for one directory and no table for the other"